unified_targets/configs/SPRACINGH7EXTREME.config

```diff
@@ -0,0 +1,236 @@
+# Betaflight / SPRACINGH7EXTREME (SP7E) 4.1.0 Jul 10 2019 / 21:50:29 (fe9d4fcc2) MSP API: 1.42
+
+board_name SPRACINGH7EXTREME
+manufacturer_id SPRO
+
+# resources
+resource BEEPER 1 D07
+resource MOTOR 1 A00
+resource MOTOR 2 A01
+resource MOTOR 3 A02
+resource MOTOR 4 A03
+resource MOTOR 5 B06
+resource MOTOR 6 B07
+resource MOTOR 7 C06
+resource MOTOR 8 C07
+resource SERVO 1 NONE
```

Comment:
The lines containing `NONE` in here should be removed.

Reply:
done